pysumma/pysumma/Simulation.py

```diff
@@ -0,0 +1,101 @@
+#from Decisions import Decisions         # This is for testing in cmd window.
+from .Decisions import Decisions       # This is for testing in this python code.
+import subprocess
+import os
+class Simulation:
+    executable = ''
+    run_suffix = '_'
+    def __init__(self, filepath):
+        self.filepath = os.path.abspath(filepath)
+        self.setting_path = file_manager_option('setting_path', self.filepath)
+        self.input_path = file_manager_option('input_path', self.filepath)
+        self.output_path = file_manager_option('output_path', self.filepath)
+        self.decision_path = file_manager_option('decision', self.filepath)
+        self.meta_time = file_manager_option('meta_time', self.filepath)
+        self.meta_attr = file_manager_option('meta_attr', self.filepath)
+        self.meta_type = file_manager_option('meta_type', self.filepath)
+        self.meta_force = file_manager_option('meta_force', self.filepath)
+        self.meta_localpar = file_manager_option('meta_localpar', self.filepath)
+        self.OUTPUT_CONTROL = file_manager_option('OUTPUT_CONTROL', self.filepath)
+        self.meta_index = file_manager_option('meta_index', self.filepath)
+        self.meta_basinpar = file_manager_option('meta_basinpar', self.filepath)
+        self.meta_basinvar = file_manager_option('meta_basinvar', self.filepath)
+        self.local_attr = file_manager_option('local_attr', self.filepath)
+        self.local_par = file_manager_option('local_par', self.filepath)
+        self.basin_par = file_manager_option('basin_par', self.filepath)
+        self.forcing_list = file_manager_option('forcing_list', self.filepath)
+        self.initial_cond = file_manager_option('initial_cond', self.filepath)
+        self.para_trial = file_manager_option('para_trial', self.filepath)
+        self.output_prefix = file_manager_option('output_prefix', self.filepath)
+        self.decision_obj = Decisions(self.setting_path.value + self.decision_path.value)
+    def execute(self):
+        if self.executable == '':
+            raise ValueError('No executable defined. Set as "executable" attribute of Simulation')
+        else:
+            cmd = "{} -p never -s {}       -m {}".format(self.executable, self.run_suffix, self.filepath)
+            subprocess.run(cmd, shell=True)
+
+class file_manager_option:
```

Comment:
I have the same general concerns about this class as with the `option` class, that we may be reading the same file many times when once would suffice.